scripts/locale-kit.ts

```diff
@@ -0,0 +1,108 @@
+import { promises as fs } from 'fs'
+import _ from 'lodash'
```

Comment:
This pkg doesn't list in the package.json use `lodash-es` instead?

Reply:
the is run in node.